superset/superset/models/core.py

```diff
@@ -685,6 +686,11 @@ def export_dashboards(cls, dashboard_ids):
                 datasource_class = copied_datasource.__class__
                 for field_name in datasource_class.export_children:
                     field_val = getattr(eager_datasource, field_name).copy()
+                    sort_by = None
+                    if len(field_val) > 0:
+                        sort_by = field_val[0].export_ordering
```

Comment:
Is `field_val[0]` always an object of the type for which you added `export_ordering`? So `SqlaTable`, `BaseMetric`, `BaseColumn`. And if `len(field_val) > 0` are all he elements of the same type?

Reply:
I've just seen `ImportMixin` so I guess that `field_val[0]` is always an instance of a class extending `ImportMixin`.

Reply:
yes. It's always an instance of a class with `ImportMixin` and always the same type in the list.